htdocs/product/class/html.formproduct.class.php

```diff
@@ -87,13 +88,18 @@ function loadWarehouses($fk_product=0, $batch = '', $fk_product_batch=0, $sumSto
 			if (!empty($batch)) 
             {
                 $sql.= " LEFT JOIN ".MAIN_DB_PREFIX."product_batch as pb on pb.fk_product_stock = ps.rowid AND pb.batch = '".$batch."'";
-            } else if (!empty($fk_product_batch))
-			{
-				$sql.= " LEFT JOIN ".MAIN_DB_PREFIX."product_batch as pb on pb.fk_product_stock = ps.rowid AND pb.rowid = '".$fk_product_batch."'";
-			}
```

Comment:
I introduced the fk_product_batch for the multiwarehouse shipments new in 4.0, but is never used. Now you introduced the llx_product_lot table, batch will the key to get lot/serial info and will be enough. But I see there is an issue when sumstock is true and batch is set. Should I also remove batch (also not used and new for 4.0), and re-introduce it in future lot/serial improvements.